pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotSegmentRestletResource.java

```diff
@@ -163,17 +163,21 @@
   @GET
   @Produces(MediaType.APPLICATION_JSON)
   @Path("/segments/{tableName}")
-  @ApiOperation(value = "List all segments", notes = "List all segments")
+  @ApiOperation(value = "List all segments. The 'liveOnly' parameter is used to get the list of segments"
+      + " which can be queried from the table. The value is false by default.", notes = "List all segments")
   public List<Map<TableType, List<String>>> getSegments(
       @ApiParam(value = "Name of the table", required = true) @PathParam("tableName") String tableName,
-      @ApiParam(value = "OFFLINE|REALTIME") @QueryParam("type") String tableTypeStr) {
+      @ApiParam(value = "OFFLINE|REALTIME") @QueryParam("type") String tableTypeStr,
+      @ApiParam(value = "Whether to return live only segments, which is a set of segments that can be queried"
```

Comment:
This is a little bit vague to me. Here essentially we are excluding the segments already replaced using the lineage info, and not excluding the segments that are either in ERROR state or not served by the servers. Suggest updating the description/method name to make that clear

Reply:
That's a good point! I'll polish the description to make it clear in the next push. In the future, if we really want to exclude the segment in ERROR state, we can also bring up another param like `excludeErrorSegments` and the like.

Reply:
Can you change the description so that this can be in sync with the new parameter name (`excludeReplacedSegments`)?

Reply:
Description updated.